tools/depends/target/python3/darwin_embedded.patch

```diff
@@ -197,6 +197,7 @@
      if (pid == 0) {
          /* Child process */
          /*
+
```

Comment:
Why is this line needed?

Reply:
was done by the diff command creating the patch

Reply:
It for sure wasn't, the above hunk applies cleanly without this additional line.